### DISCUSSION TITLE
Tag based flat organization or hierarchical folder based organization

### DISCUSSION THREAD

**Discussion by lamyergeier** [floccusaddon/floccus#1665] [Q&A]
Does it use "Tag based flat organization or hierarchical folder based organization"?

**Comment by marcelklehr** [comment]
Floccus currently only syncs folders as no browser currently supports tags *and* allows extensions to access them.

**Reply by lamyergeier** [reply]
@marcelklehr  Firefox supports tags

![image](https://github.com/floccusaddon/floccus/assets/42092626/7a853da4-e13c-4fef-b8b4-a583401ce053)

**Reply by marcelklehr** [reply]
Read the second part of my reply ;)
